server/block/sea_pickle.go

```diff
@@ -0,0 +1,165 @@
+package block
+
+import (
+	"github.com/df-mc/dragonfly/server/block/cube"
+	"github.com/df-mc/dragonfly/server/item"
+	"github.com/df-mc/dragonfly/server/world"
+	"github.com/go-gl/mathgl/mgl64"
+	"math"
+	"math/rand"
+)
+
+// SeaPickle is a small stationary underwater block that emits light, and is typically found in colonies of up to
+// four sea pickles.
+type SeaPickle struct {
+	empty
+	transparent
+
+	// ClusterCount is the amount of additional sea pickles clustered together.
+	ClusterCount int
+	// Alive is whether the sea pickle is alive.
```

Comment:
When is it alive? What does it mean practically?

Reply:
(don't tell me, I know, but documenting it might be a good idea :P)

Reply:
It might also be more consistent to make this field `Dead` as that's the convention with coral too.